Pipeline/controllers/PipelineController.py

```diff
@@ -32,10 +27,15 @@ def pipeline_file(arguments, files):
 
         bp_data = bp_result.json()
         motifs = bp_data.get("motif_graphs")
+        bp_data.pop("motif_graphs", None)
 
-        # TODO, pipelining
 
-        bp_data.pop("motif_graphs", None)
+        if rnamigos == 1:
+            rnamigos_result = RnaMigos.rnamigos_string(str(motifs))
```

Comment:
Verify that RNAMigos returns a 200.

You can do if (rnamigos_result) (true for successes)

Reply:
Agreed - solved for both pipeline_file and pipeline_string endpoints. I'm curious as to if I should also handle an error status such as 400. I mean something that would make it possible for the pipeline caller to know that the error comes from rnamigos.

Reply:
For now I just added an if result.status == 200 but maybe once we have a fully working pipeline we could figure out our approach for error handling/logging